javascript/javascript2/week3/homework/the-fastest-presser-in-this-realm/the-fastest-presser-in-this-realm.js

```diff
@@ -0,0 +1,128 @@
+const defaultGameDuration = 30;
+const startButton = document.getElementById("startButton");
+const lElement = document.getElementById("lElement");
+const lCountElement = document.getElementById("lCountElement");
+const sElement = document.getElementById("sElement");
+const sCountElement = document.getElementById("sCountElement");
+const timerElement = document.querySelector(".timer");
+const messageElement = document.getElementById("messageElement");
+const messageImageElement = document.getElementById("messageImageElement");
+
+let blowingAudio = new Audio('audio/blowing.mp3');
+let popAudio = new Audio('audio/pop.mp3');
+
+const lConfetti = new ConfettiGenerator({ target: 'lCanvas' });
+const sConfetti = new ConfettiGenerator({ target: 'sCanvas' });
+
+function quiverLetters() {
+    sElement.style.fontWeight = Math.floor(Math.random() * 8) * 100;
+    sElement.style.transform = "rotate(" + (Math.floor(Math.random() * 20) - 10) + "deg)";
+    lElement.style.fontWeight = Math.floor(Math.random() * 8) * 100;
+    lElement.style.transform = "rotate(" + (Math.floor(Math.random() * 20) - 10) + "deg)";
+}
+
+let countDownTimeOut;
+function countDown() {
+    gameTimeLeft--;
+    timerElement.innerHTML = gameTimeLeft;
+    if (gameTimeLeft > 0) {
+        countDownTimeOut = setTimeout(countDown, 1000);
+    }
+    else {
+        gameStarted = false;
+        messageElement.innerHTML = "Time's Up!";
+        if (lPressCount >= sPressCount) {
+            lConfetti.render();
+            lElement.innerHTML = ".";
+        }
+        if (lPressCount <= sPressCount) {
+            sConfetti.render();
+            sElement.innerHTML = ".";
+        }
```

Comment:
Do wee need to deal with the case where `lPressCount === sPressCount`?
```suggestion
        if (lPressCount >= sPressCount) {
            lConfetti.render();
            lElement.innerHTML = ".";
        }
        else if (lPressCount <= sPressCount) {
            sConfetti.render();
            sElement.innerHTML = ".";
        }
        else {
            // TODO
        }
```

Reply:
HI Cameron, Thank you for the suggestion.

Reply:
Thanks, but equal is handled in both if statements (in <= and >= with no else). This results in both gamers winning in case of a draw result and rendering confetti for both sides.